backend/controllers/quote.api.ts

```diff
@@ -40,5 +40,24 @@ export const quoteController = {
 		} catch (err) {
 			return next(err);
 		}
+	},
+
+	getRandomQuote: async (_req: Request, res: Response, next: NextFunction) => {
+		try {
+			const count = await QuoteModel.estimatedDocumentCount();
+
+			if (count === 0) {
+				return res.status(404).json({ message: "No quotes found" });
```

Comment:
Please use `HttpCode` enum instead of hardcoding the HTTP code. Other than that everything's looking good.

Reply:
Alright, boss! I've already fixed it!